### DISCUSSION TITLE
为啥在Edge应用商店下架了

### DISCUSSION THREAD

**Discussion by verygoodlee** [BewlyBewly/BewlyBewly#859] [Q&A] [answered]
突然发现Edge应用商店的没了，然后看到这里也改成Chrome链接了 BewlyBewly/BewlyBewly@8b85499，是开发者主动下架的吗。

虽然Edge商店的版本落后一点，但是用着比较省心，
谷歌应用商店在国内需要梯子，手动安装zip包的话更新又比较麻烦，每次出新版都要重装。

**Comment by hakadao** [comment, marked as answer]
![image](https://github.com/BewlyBewly/BewlyBewly/assets/33394391/3f19bc82-ffea-4cc2-b819-c8668b623887)
![image](https://github.com/BewlyBewly/BewlyBewly/assets/33394391/0571af76-eebd-4b63-b892-af00f2532f2a)

**Reply by hakadao** [reply]
edge 審覈強制7個工作日問題和 edge publish api 經常隔一段時間報 500 錯誤, 這些 github 查都可以查到 (我這裏懶得貼出相應問題或討論), api 500 問題是壓死我最後一根稻草
# 別問我爲何有其他解決方法爲何不想想, 現在已經對 edge 徹底死心

**Comment by hakadao** [comment]
edge 用戶提問時總是提重複問題, 且有些標題直接寫個 [BUG], 這裏只是選取部分, bilibili 私信也有一堆, 大概解釋一下之前的情況
![image](https://github.com/BewlyBewly/BewlyBewly/assets/33394391/8e241bdf-7bb8-4b1a-8e83-b6afab07499a)
![image](https://github.com/BewlyBewly/BewlyBewly/assets/33394391/52b865ac-cf25-43f7-b75c-d89386a4820e)
![image](https://github.com/BewlyBewly/BewlyBewly/assets/33394391/5864f590-57d4-49da-a30f-9ac1fc07fb9e)
![image](https://github.com/BewlyBewly/BewlyBewly/assets/33394391/fac1b10c-6c4d-4b51-bfe3-041435eecf30)

**Comment by TC999** [comment]
墙内可用 Chrome 商店：[CRX搜搜的链接](https://www.crxsoso.com/webstore/detail/bbbiejemhfihiooipfcjmjmbfdmobobp)

**Comment by xianhongtao** [comment]
需要翻墙下载实在是太糟心了，加上傻逼M$还会隔一段时间弹窗要求用户关闭第三方扩展，球球作者重新上架吧

**Reply by hakadao** [reply]
> 需要翻墙下载实在是太糟心了，加上傻逼M$还会隔一段时间弹窗要求用户关闭第三方扩展，球球作者重新上架吧

我每次和不明真相的edge用戶解釋爲何那麼久才更新更煩，明明那時readme和github提問問題模板都寫得很清楚了

**Reply by TC999** [reply]
Edge 用户也能安装 Chrome 商店的插件而且不会弹窗，墙内可用：[CRX搜搜镜像](https://www.crxsoso.com/webstore/detail/bbbiejemhfihiooipfcjmjmbfdmobobp)

**Reply by aCupofTaylor** [reply]
[CRX搜搜镜像](https://www.crxsoso.com/webstore/detail/bbbiejemhfihiooipfcjmjmbfdmobobp)上chrome插件可以下载并正常运行，作者可以直接把chrome的插件上架吗

**Reply by hakadao** [reply]
> [CRX搜搜镜像](https://www.crxsoso.com/webstore/detail/bbbiejemhfihiooipfcjmjmbfdmobobp)上chrome插件可以下载并正常运行，作者可以直接把chrome的插件上架吗

chrome web store有啊，還是一直都有找找 BewlyBewly 不就出來了

**Reply by aCupofTaylor** [reply]
就是直接将chrome store的文件上架到edge上

**Reply by hakadao** [reply]
> 就是直接将chrome store的文件上架到edge上

上面的討論沒有看到嗎？edge那邊無論如何都要7天審覈沒得快那次經常有很多傻逼b站私信騷擾我爲何edge那麼慢更新是否針對edge，爲何edge的錯那麼慢都不修，加上github也有一堆傻逼在問題模板說明了edge更新慢的情況下亂寫issue影響項目維護
只能說這一切是edge用戶爭取過來的

**Reply by aCupofTaylor** [reply]
> 上面的討論沒有看到嗎？edge那邊無論如何都要7天審覈沒得快那次經常有很多傻逼b站私信騷擾我爲何edge那麼慢更新是否針對edge，爲何edge的錯那麼慢都不修，加上github也有一堆傻逼在問題模板說明了edge更新慢的情況下亂寫issue影響項目維護 只能說這一切是edge用戶爭取過來的

抱歉，作者别激动
